src/amo/pages/Feedback/index.js

```diff
@@ -0,0 +1,478 @@
+
+    return {
+      ...defaultFormValues,
+      email: email || '',
+      name: name || '',
+    };
+  }
+
+  preventSubmit(): boolean {
+    const { loading } = this.props;
+    const { text, category } = this.state;
+
+    return loading || !category || !text || (text && text.trim() === '');
+  }
+
+  render(): React.Node {
+    const {
+      _config,
+      errorHandler,
+      i18n,
+      loading,
+      currentUser,
+      addonIdentifier,
+      abuseReport,
+    } = this.props;
+
+    if (!_config.get('enableFeatureFeedbackForm')) {
+      return <NotFoundPage />;
+    }
+
+    let errorMessage;
+
+    if (errorHandler.hasError()) {
+      log.warn(`Captured API Error: ${errorHandler.capturedError.messages}`);
+
+      if (errorHandler.capturedError.responseStatusCode === 404) {
+        return <NotFoundPage />;
+      }
+
+      errorMessage = errorHandler.renderError();
+    }
+
+    const submitButtonText = loading
+      ? i18n.gettext('Submitting your report…')
+      : i18n.gettext('Submit report');
+
+    const categories = getCategories(i18n);
+    const categoryInputs = {};
+    // eslint-disable-next-line guard-for-in
+    for (const categoryType in categories) {
+      categoryInputs[categoryType] = [];
+      categories[categoryType].forEach((category) => {
+        categoryInputs[categoryType].push(
+          <li className="Feedback-category--li" id={`li-${category.value}`}>
+            <input
+              type="radio"
+              className="Feedback-catgeory"
+              id={`feedbackCategory${category.value}`}
+              name="category"
+              onChange={this.onFieldChange}
+              value={category.value}
+              selected={this.state.category === category.value}
+            />
+            <label
+              className="Feedback--label"
+              htmlFor={`feedbackCategory${category.value}`}
+            >
+              {category.label}
+            </label>
+            <p className="Feedback--help">{category.help}</p>
+          </li>,
+        );
+      });
+    }
+
+    const abuseSubmitted = abuseReport && abuseReport.message !== undefined;
+
+    return (
+      <Page>
+        <div className="Feedback">
+          <Helmet>
+            <title>
+              {i18n.sprintf(
+                i18n.gettext('Submit Feedback for %(type)s %(identifer)s'),
+                {
+                  type: 'Addon',
+                  identifer: addonIdentifier,
+                },
+              )}
+            </title>
+          </Helmet>
+
+          {abuseSubmitted && (
+            <div>
+              <Card
+                header={i18n.gettext('You sent a report about this add-on')}
+                className="Feedback--Card"
+              >
+                <p className="ReportAbuseButton-first-paragraph">
+                  {i18n.gettext(
+                    `We have received your report. Thanks for letting us know.`,
```

Comment:
This looks a bit broken because the card doesn't take the full width.

![image](https://github.com/mozilla/addons-frontend/assets/217628/54f33b0d-c3e0-4417-b7cb-022ded38e01b)